models/models.js

```diff
@@ -5,3 +5,19 @@ exports.selectTopics = async () => {
   const { rows } = await db.query(`SELECT * FROM topics;`);
   return rows;
 };
+
+exports.selectArticleById = async (id) => {
+  try {
+    const { rows: article } = await db.query(
+      `SELECT *FROM articles 
+             WHERE article_id = $1;`,
+      [id]
+    );
+    if (article.length === 0) {
+      return Promise.reject({ status: 404, msg: "not found" });
+    }
+    return article;
+  } catch (error) {
+    throw error;
```

Comment:
Make sure you use `next(error)` instead of throw.

Reply:
I couldn't get next(error) to work, should it be a straight swap? Also thanks for answering my comment questions : )